web_client/panels/ZoomWidget.js

```diff
@@ -88,7 +88,7 @@ var ZoomWidget = Panel.extend({
     setViewer(viewer) {
         var geo = window.geo;
         var range;
-        this.viewer = viewer;
+        this.viewer = viewer.viewer;
```

Comment:
I find this mildly confusing -- in the ZoomWidget, `this.viewer` is the geojs map, whereas in the other widgets it is the viewer widget.  Perhaps for clarity it should always be the viewer widget and elsewhere in the ZoomWidget you could use `this.viewer.viewer`.  Alternately, document the result of `setViewer` in different widgets, since it differs.